Userland/Applications/PixelPaint/CloneTool.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2021, Mustafa Quraish <mustafa@cs.toronto.edu>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "CloneTool.h"
+#include "ImageEditor.h"
+#include "Layer.h"
+#include <LibGUI/Action.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/CheckBox.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/RadioButton.h>
+#include <LibGUI/ValueSlider.h>
+#include <LibGfx/Bitmap.h>
+
+namespace PixelPaint {
+
+CloneTool::CloneTool()
+{
+}
+
+CloneTool::~CloneTool()
+{
+}
```

Comment:
Since those dont do anything, just add `CloneTool() = default` and `~CloneTool() = default` to the header and remove those here.